src/Workspaces/Core/Portable/Utilities/Documentation/XmlDocumentationProvider.cs

```diff
@@ -77,11 +77,7 @@ protected override string GetDocumentationForSymbol(string documentationMemberID
                         foreach (var e in doc.Descendants("member"))
                         {
                             if (e.Attribute("name") != null)
-                            {
-                                using var reader = e.CreateReader();
-                                reader.MoveToContent();
-                                comments[e.Attribute("name").Value] = reader.ReadInnerXml();
-                            }
+                                comments[e.Attribute("name").Value] = e.ToString();
```

Comment:
https://github.com/dotnet/roslyn/commit/1bf6bfc2b23d2934f3a124b9180f703cd3a72fe6

I wonder what the perf impact of this is.

Reply:
It looks like the underlying implementation (at least in .NET Framework) is to create a string writer and then write the string out to it. Hard to know without making a microbenchmark here but it won't be as bad as that string.Concat.